wp-content/themes/Belladonna/scss/main.scss

```diff
@@ -309,7 +310,7 @@ body.home {
 		}
 	}
 	#site_title {

```

Comment:
Try to never use IDs for styling.

Reply:
I second this. _Never_ use ids for styling.